Documentation/Usage.md

```diff
@@ -696,6 +696,22 @@ A C language target is built with following flags in release mode:
 
 Swift Package Manager includes a build system that can build for macOS and Linux.
 Xcode 11 integrates with `libSwiftPM` to provide support for iOS, watchOS, and tvOS platforms.
+To build your package with Xcode from command line you can use 
+[`xcodebuild`](https://developer.apple.com/library/archive/technotes/tn2339/_index.html).
+An example invocation would be:
+
+```
+xcodebuild -scheme Foo -destination 'platform=iOS Simulator,OS=13.5,name=iPhone 11 Pro'
```

Comment:
I strongly suggest using a generic device in the example invocation here, so that we don't have to keep updating the example with specific devices (i.e. "iPhone 11 Pro running iOS 13.5") over time. You can use the following:

    xcodebuild -scheme Foo -destination 'generic/platform=iOS'
    xcodebuild -scheme Foo -destination 'generic/platform=iOS Simulator'